{
  "description": "In Lib/tarfile.py in Python through 3.8.3, an attacker is able to craft a TAR archive leading to an infinite loop when opened by tarfile.open, because _proc_pax lacks header validation.",
  "releases": {
    "buster": {
      "repositories": {
        "buster": "3.7.3-2+deb10u1"
      },
      "status": "open",
      "urgency": "not yet assigned"
    }
  },
  "scope": "local"
}